docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -38,7 +38,7 @@ A software-defined asset includes the following:
 | Name                                  | Description                                                                                                                                                                                                                          |
 | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | <PyObject object="asset" decorator /> | A decorator used to define assets.                                                                                                                                                                                                   |
-| <PyObject object="SourceAsset" />     | A class that describes an asset, but doesn't define how to compute it. <PyObject object="SourceAsset" />s are used to represent assets that other assets or jobs depend on, in settings where they can't be materialized themselves. |
+| <PyObject object="SourceAsset" />     | A class that describes an asset, but doesn't define how to compute it. <PyObject object="SourceAsset" />s are used to represent assets that other assets depend on, in settings where they can't be materialized themselves. |
```

Comment:
removed "or jobs" since it didn't seem necessary and references a concept the user may not have familiarity with